versions/3.0.md

```diff
@@ -1124,6 +1100,185 @@ application/json:
 
 ```
 
+##### Considerations for file uploads
+
+In contrast with the 2.0 specification, describing `file` input/output content in OpenAPI is 
+described with the same semantics as any other schema type. Specifically:
+
+```yaml
+# content transferred with base64 encoding
+schema:
+  type: string
+  format: base64
```

Comment:
This is `byte` today. We can change, but needs to be updated in the other section as well.